lib/phlex/csv.rb

```diff
@@ -48,43 +55,56 @@ def content_type
 
 	def column(header = nil, value)
 		if @_first
-			@_headers << header
+			@_headers << escape(header)
 		elsif header != @_headers[@_current_column_index]
 			raise "Inconsistent header."
 		end
 
-		@_current_row << value
+		@_current_row << escape(value)
 		@_current_column_index += 1
 	end
 
 	def each_item(&block)
 		collection.each(&block)
 	end
 
-	def collection_yielder(record)
+	def yielder(record)
```

Comment:
Unrelated to the main purpose of this PR, I think I prefer `yielder` for this method, since it's not really yielding the collection, it's really yielding a single item in the collection.